alf/algorithms/sac_algorithm.py

```diff
@@ -531,26 +552,52 @@ def _predict_action(self,
         return action_dist, action, q_values, new_state
 
     def predict_step(self, inputs: TimeStep, state: SacState):
-        action_dist, action, _, action_state = self._predict_action(
+        action_dist, action, q_values, action_state = self._predict_action(
             inputs.observation,
             state=state.action,
             epsilon_greedy=self._epsilon_greedy,
             eps_greedy_sampling=True)
+        info = SacInfo(action_distribution=action_dist)
+        if (alf.summary.render.is_rendering_enabled()
```

Comment:
This block of rendering code should be kept as a local change as it's not essential to SAC.